### ISSUE TITLE
Race condition with regional profiles

### ISSUE THREAD

**Issue by xylar** [MPAS-Dev/MPAS-Analysis#763]
From @golaz:

@xylar : I ran a complete post-processing overnight test using the new E3SM Unified test environment on a 110 year coupled simulation. The post-processing included 38 jobs sampling a good fraction of all the tools (nc climatologies, time series, E3SM Diags, MPAS-Analysis). All jobs completed successfully except one.

The failure was in the second iteration of MPAS-Analysis. Here is the error

oceanRegionalProfiles_combineOceanBasinsProfiles_0001-0100.log

`FileNotFoundError: [Errno 2] No such file or directory: b'/compyfs/gola749/E3SM_simulations/20201202.alpha5_59_target.piControl.ne30pg2_r05_EC30to60E2r2-1900_ICG.compy/post/analysis/mpas_analysis/output/timeseries/OceanBasins/regionalProfiles_OceanBasins_0073-0073.nc'`

Surprisingly, the file actually exists, but it looks like it was created **one minute after** the log file. Has anything changed in MPAS-Analysis that could create a race condition? Unless, it's just the compyfs being flaky.

**Comment by golaz** [comment]
@xylar : I ran into this issue at least twice using the new E3SM unified test environment. I then switched back to the latest environment without any issue so far.

**Comment by xylar** [comment]
Thanks @golaz.  I'll try to fix this in the next few days so we can re-test before the Jan 1st release

**Comment by xylar** [comment]
@golaz, the original directory where this happened is gone.  Could you share an example where it happened?  Specifically, it would be very helpful to have the log files and your config file(s).

**Comment by xylar** [comment]
@golaz, this is not a fix to the race condition but I noticed that you have `oceanRegionalProfiles` disabled by default here:
https://github.com/E3SM-Project/PreAndPostProcessingScripts/blob/master/postprocessing_bundle/v2/templates/default.ini#L96
We separated out a new analysis task, `hovmollerOceanRegions` in the v1.2.9 release that is part of the E3SM-Unified test environment, and it would seem that you probably would want that disabled as well, given that it would have been disabled as part of `oceanRegionalProfiles` in previous MPAS-Analysis versions.  So you will likely want to add `no_hovmollerOceanRegions` to that line of `default.ini`.  This flag will just be ignored by earlier versions of MPAS-Analysis, where this task doesn't exist.
